packages/core/src/components/essentials/tooltip.ts

```diff
@@ -80,10 +85,12 @@ export class Tooltip extends Component {
 				}
 
 				// Position the tooltip
-				this.positionTooltip(e);
+				if (this.isTooltipEnabled) {
```

Comment:
we probably also wouldn't want to call `getTooltipHTML` or run the logic above if tooltip is disabled

Reply:
please see the updated code